version.go

```diff
@@ -0,0 +1,59 @@
+package venova
+
+import (
+	"log"
+	"runtime"
+)
+
+type VersionInfo struct {
+	Name      string
+	Version   string
+	Revision  string
+	Reference string
+	GoVers    string
+	BuiltAt   string
+	OS        string
+	Arch      string
+}
+
+var (
+	NAME      = "venova"
+	NAME2     = "venova-stream"
+	VERSION   = "main"
+	REVISION  = "HEAD"
+	REFERENCE = "HEAD"
+	GoVers    = runtime.Version()
+	BUILT     = "now"
+	OS        = runtime.GOOS
+	Arch      = runtime.GOARCH
+	Version   VersionInfo
+)
+
+func GetVersion(customName string) VersionInfo {
```

Comment:
This function is super ugly:

- Should be called `GetVersionInfo(...)`
- Should be a pure function (`Get` almost _always_ implies purity), I don't see the point of `GetVersion()` and also exporting `Version`.